json_serializable/test/custom_configuration_test.dart

```diff
@@ -19,7 +19,7 @@ import 'shared_config.dart';
 
 LibraryReader _libraryReader;
 
-void main() async {
+Future main() async {
```

Comment:
Ya, I would remove this lint lol these do not provide value

Reply:
ack